host/builder.go

```diff
@@ -106,9 +104,8 @@ func BuildNamedHost(priv_key crypto.PrivKey) core.Host {
 		listenAddrs = libp2p.ListenAddrs()
 	}
 	host, err := libp2p.New(
-		context.Background(),
 		listenAddrs,
-		libp2p.EnableRelay(circuit.OptHop),
+		libp2p.EnableRelay(),
```

Comment:
Why do we remove this option?

Reply:
I have removed this to sync with the latest interface of `libp2p.EnableRelay()`